.github/workflows/pr-build.yml

```diff
@@ -0,0 +1,76 @@
+name: build and test
+on:
+  push:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  pull_request:
+    branches:
+      - main
+    paths-ignore:
+      - '**/*.md'
+  create:
+  pull_request_target:
```

Comment:
Does this replace `pull_request` or is it in addition to? if it's in addition to, it seems like it should have the same filter rules

Reply:
I removed filters from both. And yes, it's in addition to it. So `pull_request` will fail on forks but `pull_request_target` will pass